README.md

```diff
@@ -1,117 +1,84 @@
-[![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=crossplaneio_crossplane&metric=reliability_rating)](https://sonarcloud.io/dashboard?id=crossplaneio_crossplane)
-[![Security Rating](https://sonarcloud.io/api/project_badges/measure?project=crossplaneio_crossplane&metric=security_rating)](https://sonarcloud.io/dashboard?id=crossplaneio_crossplane)
+[Crossplane] is the open source multicloud control plane. With Crossplane you
+can manage your applications and infrastructure across clouds, regions, and
+clusters. Crossplane is composable and Kubernetes native. Deploy it to an
+existing Kubernetes cluster to manage cloud infrastructure like databases,
+buckets, and caches using `kubectl`. Deploy it standalone to manage Kubernetes
+clusters, the workloads running on them, and the cloud infrastructure those
+workloads depend upon.
 
-# Overview
+## Architecture
 
-Crossplane is an open source multicloud control plane to manage your
-cloud-native applications and infrastructure across environments, clusters,
-regions and clouds. It enables provisioning and full-lifecycle management
- of applications and managed services from your choice of cloud using `kubectl`.
+![Architecture diagram](docs/media/crossplane-overview.png)
 
-Crossplane can be installed into an existing Kubernetes cluster to add managed
-service provisioning or deployed as a dedicated control plane for multi-cluster
-management and workload scheduling.
+Crossplane builds on the Kubernetes control plane. It is composed of [Stacks] -
+easy to install packages of Kubernetes [custom resources and controllers] that
+extend Crossplane with new functionality. A stack can extend Crossplane with the
+ability to manage the infrastructure of a new cloud provider, or to deploy and
+manage a new cloud-native application. This enables Crossplane users to:
 
-Crossplane enables the community to build and publish Stacks to add more clouds
-and cloud services to Crossplane with support for out-of-tree extensibility and
-independent release schedules. Crossplane includes Stacks for [GCP][stack-gcp], 
-[AWS][stack-aws], and [Azure][stack-azure] today.
+1. Install the **infrastructure stacks** for their desired clouds to enable
+   on-demand, **cloud-portable provisioning of infrastructure** services like
+   databases, caches, and Kubernetes clusters.
+1. Define **portable application workloads**, including their cloud
+   infrastructure dependencies, and schedule them to the most appropriate
+   infrastructure across any cloud.
+1. Package their workloads as an **application stack** that others may easily
+   deploy.
 
-<h4 align="center"><img src="docs/media/crossplane-overview.png" alt="Crossplane"></h4>
+The full vision and architecture of the Crossplane project is described in our
+[architecture document].
```

Comment:
Took a read through this again, and while it has some valuable information about the motivation for Crossplane, it contains a lot of outdated information about the mechanics of the system. How would you feel about omitting this until we have that document updated? Or, alternatively, we could update that document before merging this (happy to help out with that if needed!).

Reply:
I think there may be value in adding a note to the architecture document reflecting that there have been some changes since we first released Crossplane and directing folks to our design documents. I would prefer not to block this PR on doing so, given `README.md` already links to the architecture document.